### PR TITLE
[ADD] install: geolocalization

### PR DESCRIPTION
**Pull request by masp-odoo**
taskid-3694395

### CODE REVIEW

File: content/administration/install/geolocalization.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+:show-content:
```
[the remark this answers is not in the document]
Comment (masp-odoo) [reply content/administration/install/geolocalization.rst:1]: Indeed I did not write it anywhere else but this day it seems that I felt like it... Thanks!

File: content/administration/install/geolocalization.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+===============
+Geolocalization
```
[the remark this answers is not in the document]
Comment (masp-odoo) [reply content/administration/install/geolocalization.rst:4]: Ok thanks for the explanation.

File: content/administration/install/geolocalization.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+For instance, the routes in the Field Service App.
+
+  .. image:: geolocalization/field-service.png
+     :alt: Field Service App
+
+You can also locate all your contacts on a map.
+
+  .. image:: geolocalization/contacts.png
+     :alt: Contacts on a map
```
[the remark this answers is not in the document]
Comment (masp-odoo) [reply content/administration/install/geolocalization.rst:18]: So should I completely remove one of the examples?

File: content/administration/install/geolocalization.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Odoo provides two third-party providers to do so.
+
+Open Street Map
```
[the remark this answers is not in the document]
Comment (masp-odoo) [reply content/administration/install/geolocalization.rst:22]: And I agree, I bolded each options.

File: content/administration/install/geolocalization.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+===============
+
+Open Street Map (OSM) is a collaborative project that creates a free, editable world map. It
+provides a platform for individuals and organizations to share, view, and use map data to create a
+detailed and up-to-date map that can be used for various purposes.
+
+If you wish to continue using the free Open Street Map on your database, it is possible to fix the
+location accuracy by fixing it in Open Street Map. Please see the following `link <https://www.openstreetmap.org/fixthemap>`_
+for more info on mapping a location.
```
[the remark this answers is not in the document]
Comment (masp-odoo) [reply content/administration/install/geolocalization.rst:31]: I agree.

**Comment by xpl-odoo** [comment]
@robodoo r+

**Comment by xpl-odoo** [comment]
@robodoo retry

**Pull request closed without merging**